### PR TITLE
[FIX] Don't report a missing module that's contained in another bundle

### PR DESCRIPTION
**Pull request by codeworrior**

### CODE REVIEW

File: lib/lbt/bundle/Resolver.js [hunk 1]
```diff
@@ -105,20 +124,29 @@ class BundleResolver {
...
+									}
+								);
+							}
+
 							if ( decomposable ) {
 								// bundles are not added, only their embedded modules
 								promises = dependencyInfo.subModules.map( (included) => {
 									return checkAndAddResource(included, depth + 1,
-										"**** error: missing submodule " + included + ", included by " + resourceName);
+										`****** error: missing submodule ${included}, included by ${resourceName}`);
```
Comment (matz3) [inline lib/lbt/bundle/Resolver.js:149]: Why the 2-⭐ upgrade?

**Review by RandomByte — COMMENTED** [review]
LGTM, just a small remark

**Review by RandomByte — APPROVED** [review]
Tested locally with self-contained build in openui5-sample-app with a dependency to latest sap.viz snapshot

**Review by matz3 — APPROVED** [review]

**Comment by RandomByte** [comment]
Merging this PR even with the failing tests as they are known false-positives.

**Pull request merged**